src/review/views.py

```diff
@@ -380,6 +380,10 @@ def in_review(request, article_id):
 
     if request.POST:
 
+        if 'move_to_review' in request.POST and article.stage == submission_models.STAGE_UNASSIGNED:
+            article.stage = submission_models.STAGE_UNDER_REVIEW
+            article.save()
```

Comment:
Should we check if there is an editor assigned as well?  This is what we do on the "Unassigned" screen.